sw.js

```diff
@@ -0,0 +1,61 @@
+---
+layout: null
+---
+'use strict';
+
+const version = 'v1';
+const staticCacheName = "version-{{site.time | date: '%Y%m%d%H%M%S'}}";
+
+addEventListener('install', installEvent => {
+     console.log('The service worker is installing...');
+     skipWaiting();
+     installEvent.waitUntil(
+       caches.open(staticCacheName)
+       .then( staticCache => {
+         return staticCache.addAll([
+           '/assets/css/style.css',
```

Comment:
The name of the stylesheet was moved to `/assets/main.css`.

Reply:
Last I touched this, it had been [changed][] in order to match Pages' "house style". 

[changed]: https://github.com/jekyll/minima/pull/305/commits/7a64517c183fc4a823ebfee599806fd9ce509284

Reply:
Ah, ok, so I had it backwards.  It got moved to `assets/css/style.css`.  That explains it.